crates/sui-core/src/transaction_manager.rs

```diff
@@ -57,8 +57,11 @@ impl TransactionManager {
             .pending_execution
             .iter()
         {
-            let cert = node_sync_store.get_cert(epoch, &digest).unwrap().unwrap();
-            pending_certs.push(cert);
+            if let Ok(Some(cert)) = node_sync_store.get_cert(epoch, &digest) {
+                pending_certs.push(cert);
+            } else {
+                warn!(tx_digest=?digest, "Certificate cannot be recovered from node_sync_store!");
```

Comment:
i think we should check if the cert was executed, and emit an error log if not.

if it was not executed, we may have lost it for some reason we weren't able to understand, and this could cause loss of liveness. But if we log the digest of the missing cert, we can probably retrieve it from some other validator, and send it to the affected validator to get things going again.